QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.Web/wwwroot/js/Scanner.js

```
@@ -0,0 +1,157 @@
+        scannerStatus.className = 'alert alert-success mt-3 w-100';
+        scannerStatus.textContent = 'Scanner active - position QR code in view';
+        statusCard.className = 'status-card status-scanning';
+        statusTitle.textContent = 'Scanner Active';
+        statusMessage.textContent = 'Ready to scan QR codes';
+        startScan();
+    } catch (err) {
+        scannerStatus.className = 'alert alert-danger mt-3 w-100';
+        scannerStatus.textContent = 'Error accessing camera: ' + err.message;
+        statusCard.className = 'status-card status-error';
+        statusTitle.textContent = 'Camera Error';
+        statusMessage.textContent = 'Unable to access camera: ' + err.message;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)